index/load_doc.ml

```diff
@@ -34,10 +34,14 @@ let cost_doc = function
     0
   | _ -> 100
 
-let cost ~name ~kind ~doc_html ~rhs ~cat =
+let cost ~name ~kind ~doc_html ~rhs ~cat ~favoured_packages =
   String.length name
   + (5 * path_length name)
-  + (if string_starts_with ~prefix:"Stdlib." name then 0 else 50)
+  + (if List.exists
+          (fun package -> string_starts_with ~prefix:(package ^ ".") name)
```

Comment:
I think user are going to be surprised by the behaviour if we don't add the dot, because its not named "favoured prefixes" but "favoured packages". Maybe we can change both ?

Reply:
Well the module name we match against isn't a package either :P For the integration with dune, how will you fill that flag?

Reply:
I do not know yet, but if I do not find a good way to get a package name I will make it empty